lib/iris/tests/test_grib_save_rules.py

```diff
@@ -1,148 +0,0 @@
-        grib = None
-        cube = iris.cube.Cube([1, 2, 3, 4, 5])
-        cube.add_aux_coord(iris.coords.AuxCoord([12345], "height", units="m"))
-
-        grib_save_rules.set_fixed_surfaces(cube, grib)
-
-        mock_set.assert_any_call(grib, "typeOfFirstFixedSurface", 103)
-        mock_set.assert_any_call(grib, "scaleFactorOfFirstFixedSurface", 0)
-        mock_set.assert_any_call(grib, "scaledValueOfFirstFixedSurface", 12345)
-        mock_set.assert_any_call(grib, "typeOfSecondFixedSurface", -1)
-        mock_set.assert_any_call(grib, "scaleFactorOfSecondFixedSurface", 255)
-        mock_set.assert_any_call(grib, "scaledValueOfSecondFixedSurface", -1)
-
-    @mock.patch.object(gribapi, "grib_set")
-    def test_no_vertical(self, mock_set):
-        grib = None
-        cube = iris.cube.Cube([1, 2, 3, 4, 5])
-        grib_save_rules.set_fixed_surfaces(cube, grib)
-        mock_set.assert_any_call(grib, "typeOfFirstFixedSurface", 1)
-        mock_set.assert_any_call(grib, "scaleFactorOfFirstFixedSurface", 0)
-        mock_set.assert_any_call(grib, "scaledValueOfFirstFixedSurface", 0)
-        mock_set.assert_any_call(grib, "typeOfSecondFixedSurface", -1)
-        mock_set.assert_any_call(grib, "scaleFactorOfSecondFixedSurface", 255)
-        mock_set.assert_any_call(grib, "scaledValueOfSecondFixedSurface", -1)
-
-
-@tests.skip_grib
-class Test_phenomenon(tests.IrisTest):
-    @mock.patch.object(gribapi, "grib_set")
-    def test_phenom_unknown(self, mock_set):
-        grib = None
-        cube = iris.cube.Cube(np.array([1.0]))
-        # Force reset of warnings registry to avoid suppression of
-        # repeated warnings. warnings.resetwarnings() does not do this.
-        if hasattr(grib_save_rules, "__warningregistry__"):
-            grib_save_rules.__warningregistry__.clear()
-        with warnings.catch_warnings():
-            # This should issue a warning about unrecognised data
-            warnings.simplefilter("error")
-            with self.assertRaises(UserWarning):
-                grib_save_rules.set_discipline_and_parameter(cube, grib)
-        # do it all again, and this time check the results
-        grib = None
-        cube = iris.cube.Cube(np.array([1.0]))
-        grib_save_rules.set_discipline_and_parameter(cube, grib)
-        mock_set.assert_any_call(grib, "discipline", 255)
-        mock_set.assert_any_call(grib, "parameterCategory", 255)
-        mock_set.assert_any_call(grib, "parameterNumber", 255)
-
-    @mock.patch.object(gribapi, "grib_set")
-    def test_phenom_known_standard_name(self, mock_set):
-        grib = None
-        cube = iris.cube.Cube(
-            np.array([1.0]), standard_name="sea_surface_temperature"
-        )
-        grib_save_rules.set_discipline_and_parameter(cube, grib)
-        mock_set.assert_any_call(grib, "discipline", 10)
-        mock_set.assert_any_call(grib, "parameterCategory", 3)
-        mock_set.assert_any_call(grib, "parameterNumber", 0)
```

Comment:
This was just removed (not moved to iris-grib)
We already have a test for this: https://github.com/SciTools/iris-grib/blob/master/iris_grib/tests/unit/save_rules/test_set_discipline_and_parameter.py#L50